packages/dropbox/dropbox.0.2/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+authors: [ "Christophe Troestler <Christophe.Troestler@umons.ac.be>" ]
+tags: ["dropbox" "sync"]
+license: "LGPL-3.0 with OCaml linking exception"
+homepage: "https://github.com/Chris00/ocaml-dropbox"
+dev-repo: "git+https://github.com/Chris00/ocaml-dropbox.git"
+bug-reports: "https://github.com/Chris00/ocaml-dropbox/issues"
+doc: "https://Chris00.github.io/ocaml-dropbox/doc"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.1"}
```

It looks like you need at least 1.1 from the dune-project on the repo. Please correct me if I am wrong.

Reply:
No idea but there is no problem to require that minimal version.